### ISSUE TITLE
Changing Subtitle URL when playing

### ISSUE THREAD

**Issue by snappycode1** [kingslay/KSPlayer#33]
I know how to set a subtitle url when the media is first set to play()

Is it possible to change the subtitle URL once media is playing() ?

I'm implementing my own open subtitles code rather than the builtin code.

**Comment by kingslay** [comment]
yes. you can change KSPlayerResource.subtitle  once media is playing()

**Comment by snappycode1** [comment]
thanks, but there is no KSPlayerResource.subtitle option ?  am I missing something?

**Comment by kingslay** [comment]
` open func set(resource: KSPlayerResource, definitionIndex: Int = 0, isSetUrl: Bool = true) {`

**Comment by snappycode1** [comment]
Thanks, I'm now getting a different issue.

When I select an open subtitles .srt file to open I get the following :

Subtitle encoding format is not supported.

I've tried all different languages and they all come back the same

`asset?.subtitle = KSURLSubtitle(url: subtitleFileURL, encoding: .unicode)`

I've tried all sorts of encoding options but none seem to work.  Is there something I'm missing?

thanks.
